index.js

```diff
@@ -210,10 +210,13 @@ var lastResorts = [
   ['BEER', 'Beerware'],
   ['BOOST', 'BSL-1.0'],
   ['BSD', 'BSD-2-Clause'],
+  ['CDDL', 'CDDL-1.1'],
```

Comment:
@aviadatsnyk, right, it's not only npm :) I'm actually pretty new to this repo, my main connection was playing around with npm license stats (might write a blog post about it). Out of curiosity: how do you use this project with maven? Is it part of some toolkit?
Also, in my personal opinion, since you put in the effort, if this benefits you we should merge it. Would you want a new version published? In that case we need @kemitchell to publish it.